internal/gengapic/genrest.go

```diff
@@ -470,15 +470,11 @@ func (g *generator) shouldDisableComplexPaths(m *descriptor.MethodDescriptorProt
 
 	info, _ := getHTTPInfo(m)
 
-	re := regexp.MustCompile(`[^a-zA-Z0-9_.]+?`)
-	pathParams := g.pathParams(m)
-	for pName := range pathParams {
-		if re.MatchString(pName) {
-			p(`    return nil, fmt.Errorf("complex url paths are not yet supported: %s")`, info.url)
-			p("}")
-			p("")
-			return true
-		}
+	if len(g.pathParams(m)) > 1 {
```

Comment:
I added this before the regex change (above) - now I think it might not be needed.

Reply:
Agreed

Reply:
Thanks, will remove.

Reply:
Done. This function (`shouldDisableComplexPaths`) is now empty. I left it in place (returning `false`). Would you prefer that I remove it?

Reply:
Hey sorry, I got stuck in PHP land. Let's remove it and its references altogether, please.

Reply:
Done. Thanks!